src/pages/[product]/tutorials/[collectionSlug].tsx

```diff
@@ -0,0 +1,7 @@
+import CollectionView from 'views/collection-view'
+import { generateStaticFunctions } from 'views/collection-view/server'
+
+const { getStaticProps, getStaticPaths } = generateStaticFunctions()
```

Comment:
I could consider moving the definition of these functions back into this file, might give it more meaning.

Reply:
Good point! I think the `src/views/collection-view/server.ts` is a little long to totally copy/paste over here, but maybe the helpers that the generated `getStaticPaths` and `getStaticProps` functions call could be defined in a separate file? 💭

Reply:
Yeah I'm thinking of leaving the helper functions in `/collection-view/server.ts` and just bringing back `getStaticProps` & `getStaticPaths` here.